explorer/scripts/merge_issuer_csvs.py

```diff
@@ -0,0 +1,85 @@
+"""
+Manual script for merging csvs into 1 large csv per state with plan info.
+
+TODO: Incorporate this into a script with arguments.
+"""
+import pandas as pd
+
+HEALTHCARE_GOV_PATH = '/home/jovyan/work/data/healthcare_gov'
+
+state = 'MO'
+
+# Hard coded due to lack of Company info in Machine Readable PUF.
+# TODO: Automate this dictionary creation.
+issuer_dict = {'29416': 'BestLife', '32753': 'Anthem', '74483': 'Cigna'}
+
+csvs = [HEALTHCARE_GOV_PATH + '/{}/{}.csv'.format(state, issuer) for issuer in issuer_dict.keys()]
+dfs = [pd.read_csv(csv) for csv in csvs]
+
+print(csvs)
+
+for issuer_id, df in zip(issuer_dict.keys(), dfs):
+    df['IssuerId'] = int(issuer_id)
+    df['CompanyName'] = issuer_dict[issuer_id]
+
+print(dfs[0].head())
```

Comment:
No need for all these prints?
Also, maybe use logging to be consistent with your other scripts?